lib/rubocop/cop/rspec/predicate.rb

```diff
@@ -0,0 +1,182 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Prefer using predicate matcher instead of truthiness
```

Comment:
FWIW once #174 is merged this will always be the same as what is in the config file